### PR TITLE
Fix rdma-core package post scripts

### PR DESCRIPTION
**Pull request by AZaugg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] If you are adding/removing a .spec file that has multiple-versions supported, please add [@microsoft/cbl-mariner-multi-package-reviewers](https://github.com/orgs/microsoft/teams/cbl-mariner-multi-package-reviewers) team as reviewer [(Eg. golang has 2 versions 1.18, 1.21+)](https://github.com/microsoft/azurelinux/tree/2.0/SPECS/golang)
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
When installing the current package, rdma kernel modules are not installed, due to bad udevadm reference. Fixing the post script to ensure KO are loaded after package install

###### Change Log  

- Updated path to udevadm

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Will provide test details in the ADO

### CODE REVIEW

File: SPECS/rdma-core/rdma-core.spec [hunk 1]
```diff
@@ -304,10 +304,10 @@ rm -f %{buildroot}/%{_sbindir}/srp_daemon.sh
 %postun -n librdmacm -p /sbin/ldconfig
 
 %post -n rdma-core
-if [ -x /sbin/udevadm ]; then
-/sbin/udevadm trigger --subsystem-match=infiniband --action=change || true
-/sbin/udevadm trigger --subsystem-match=net --action=change || true
-/sbin/udevadm trigger --subsystem-match=infiniband_mad --action=change || true
+if [ -x /bin/udevadm ]; then
```
Comment (mfrw) [inline SPECS/rdma-core/rdma-core.spec:307]: One of the GitHub PR checks seem to be failing.
@AZaugg

**Comment by AZaugg** [comment]
Buddy build passing https://dev.azure.com/mariner-org/mariner/_build/results?buildId=628701&view=results

File: SPECS/rdma-core/rdma-core.spec (hunk 1 above)
Comment (AZaugg) [reply SPECS/rdma-core/rdma-core.spec:307]: Fixed

**Comment by eric-desrochers** [comment]
Denied by the Stable Release maintainer team as customer is satisfied with current workaround.

**Comment by eric-desrochers** [comment]
@AZaugg please close this PR.

**Pull request closed without merging**